templates/project.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# SPDX-FileCopyrightText: 2023 CERN (home.cern)
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+project:
```

Comment:
missing fields here are 1) the one to request a Discource channel and 2) the main contact email to use if we have trouble with their content.

Reply:
Are those fields somehow going to be rendered into the project page or are they just information for us?

Reply:
As discussed, we will explore other ways of storing this info.